libtransmission/announcer.cc

```diff
@@ -56,9 +56,7 @@ static auto constexpr MaxScrapesPerUpkeep = int{ 20 };
 /* how many infohashes to remove when we get a scrape-too-long error */
 static auto constexpr TrMultiscrapeStep = int{ 5 };
 
-/***
-****
-***/
+///
```

Comment:
The normal syntax for a separator line is:
```
#pragma mark -
```

A triple slash is a documentation comment and is not supposed to be empty

Reply:
I always thought that `#pragma mark` is a Xcode extension supported only by their editor, so there's nothing normal about it. E.g. VSCode doesn't support it (not that it supports collapsing sections separated by comments).

Reply:
Some suggest that newer Xcode versions support `// MARK: ...` syntax as well (or as a newer alternative?)

Reply:
`#pragma mark` is specific to Xcode, so I don't think it makes sense here. Also it has the disadvantage of looking bad in every environment except for Xcode.

I don't think there is a "right answer" here since this is all opinion ... but if `///` is triggering some warnings e.g. from a linter saying "hey this documentation comment is empty" then what do y'all think about  either


a) `// ---` (basically, a comment + a markdown hrule)

b) `////` (four slashes instead of three so that the docs linter won't be confused)

Reply:
Go for `a`: the comment + markdown.
The quadruple slash would have similar issues as a triple slash.

Reply:
Updated in fb72831